externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/operation/ExecuteStatement.scala

```diff
@@ -180,21 +180,24 @@ class ExecuteStatement(
         val sessionId = session.handle.identifier.toString
         val savePath = session.sessionManager.getConf.get(OPERATION_RESULT_SAVE_TO_FILE_DIR)
         saveFileName = Some(s"$savePath/$engineId/$sessionId/$statementId")
-        // Rename all col name to avoid duplicate columns
-        val colName = range(0, result.schema.size).map(x => "col" + x)
-
-        val codec = if (SPARK_ENGINE_RUNTIME_VERSION >= "3.2") "zstd" else "zlib"
-        // df.write will introduce an extra shuffle for the outermost limit, and hurt performance
-        if (resultMaxRows > 0) {
-          result.toDF(colName: _*).limit(resultMaxRows).write
-            .option("compression", codec).format("orc").save(saveFileName.get)
-        } else {
-          result.toDF(colName: _*).write
-            .option("compression", codec).format("orc").save(saveFileName.get)
+        val saveFile = saveFileName.get
+        FileUtil.fullyDeleteOnExit(new File(saveFile))
```

Comment:
I don't think this method can clean up remote files, such as HDFS.

Reply:
OK, changed to use hadoop fs to cleanup saved result file, as exited code in `close()`.